frontend/app/src/utils/url.ts

```diff
@@ -55,15 +55,22 @@ export const getEtherScanRegisterUrl = (location: string) => {
         external: 'https://etherscan.io/register',
         route: { path: Routes.API_KEYS_EXTERNAL_SERVICES, hash: `#${location}` }
       };
-    case TRADE_LOCATION_POLYGON_POS:
+    // TODO: remove the string modification when https://github.com/rotki/rotki/issues/6725 is resolved
+    case toSnakeCase(TRADE_LOCATION_POLYGON_POS):
       return {
         external: 'https://polygonscan.com/register',
-        route: { path: Routes.API_KEYS_EXTERNAL_SERVICES, hash: `#${location}` }
+        route: {
+          path: Routes.API_KEYS_EXTERNAL_SERVICES,
+          hash: `#${TRADE_LOCATION_POLYGON_POS}`
```

Comment:
@lukicenturi will the hash not contain space? since you used the constant:
`#polygon pos`
I think `#{location}` should be fine

Reply:
yeah it works, it will be encoded as %20
If I keep #${location}, I need to change several things on the EvmRpc page.
This is just intended as small fix, and if backend change it to use `_`, we can remove this.